### ISSUE TITLE
Random error on incoming validation net::ERR_INCOMPLETE_CHUNKED_ENCODING

### ISSUE THREAD

**Issue by lcalvy** [wprl/baucis#301]
I'm using version 1.6.3
While querying a controller that throw an error, some responses randomly have incorrect chunked encoding.

``` javascript
    controller.request(function(req, res, next) {
        req.baucis.incoming(function(context, callback) {
            if (Array.isArray(context.incoming.photos)) {
                callback(require('baucis').Error.UnsupportedMediaType('Photo format is not allowed'));
            }
            else {
                callback(null, context);
            }
        });
        next();
    });
```

![image](https://cloud.githubusercontent.com/assets/937507/11045927/97bb5262-872a-11e5-895c-aa53d000e732.png)

net::ERR_INCOMPLETE_CHUNKED_ENCODING

![image](https://cloud.githubusercontent.com/assets/937507/11045959/beea4dd4-872a-11e5-95e1-f7a25b7807c5.png)

**Comment by lcalvy** [comment]
@wprl any news?
This is a real blocking issue for our project.
Do you need more information?

**Comment by wprl** [comment]
This is what's happening where I live right now: 
http://abcnews.go.com/US/wireStory/online-threats-heighten-tensions-university-missouri-35123997

http://www.nbcnews.com/news/us-news/jonathan-butler-how-grad-students-hunger-strike-toppled-university-president-n460161

I will look into your issue as soon as possible.

**Comment by wprl** [comment]
OK, thanks for your patience.  

My heart goes out to you and your country in the devastating aftermath of the terrorist attacks in Paris.  I only wish words could express my deep feeling of sadness and anger at the terrible events.

--------------------------

Here is what I could come up with as a start for debugging this odd issue:

Have you tried any of the suggestions from http://stackoverflow.com/questions/22608564/neterr-incomplete-chunked-encoding-in-chrome-only or http://stackoverflow.com/questions/29894154/chrome-neterr-incomplete-chunked-encoding-error ?  Of particular interest is the 'Predict network actions to improve page load performance' feature of Chrome which can be turned off in chrome://settings … if the error does not occur with this turned off that would be a clue.  

I gather that for some reason the response is being cut off either at the server, network, cache, or anti-virus level.  Are there any headers sent back with the failing responses?  Does it work in other browsers or on other computers?

**Comment by lcalvy** [comment]
Hi, 

Thank you for compassion. Yes we had better days and it's a here a sad mind.

I will test that but I'm on mac OS, no antivirus, no firewall, no proxy,....
Problems only occur when I throw an exception in "req.baucis.incoming(function(context, callback) {" and never ever occur in another context. 
We used baucis in production for 1 year ago, we have end 2 end tests on every push, we never saw that before. 
First time and only time it's appear was when we have tested streaming and exception handling in incoming streaming feature.

Regards
